apps/litter-networker-api/src/app.py

```
@@ -16,16 +17,52 @@
+            request.url.path,
+            response.status_code,
+        )
+        return response
+
+    @app.exception_handler(ValueError)
+    async def value_error_handler(_request, exc: ValueError):
+        logger.warning("Handled Litter Networker API validation error: %s", exc)
+        return JSONResponse(status_code=400, content={"detail": _error_detail(exc, "Bad request")})
+
     @app.exception_handler(Exception)
     async def unhandled_exception_handler(_request, exc: Exception):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `unhandled_exception_handler`

(ANN202)